tests/yang-old/yolo/yolo-system.yang

```diff
@@ -134,4 +134,15 @@ module yolo-system {
       }
     }
   }
+
+  notification alarm-triggered {
+    description
+      "Notification about a new alarm.";
+    leaf description {
+      type string;
+    }
+    leaf severity {
+      type uint32;
+    }
+  }
```

Comment:
It would be great if you could add support for notifications in the schema diff feature.

You can validate that it works properly by removing the notification from the "old" version of the YANG module and update unit tests accordingly.

Reply:
Done! (It was easier than expected! Initially I did not pay much attention to what the diff unit test was doing...)

Reply:
I think it needs additional changes in this file

https://github.com/CESNET/libyang-python/blob/master/libyang/diff.py#L43-L45

Reply:
Sorry, this morning I just blindly made the test pass without analyzing what it was actually doing.

I checked also the function `snode_changes` (https://github.com/CESNET/libyang-python/blob/master/libyang/diff.py#L274), for which I consider there is no adaptation to be made, but I'm not 100% sure about it. Can you config this?

Reply:
Yes it looks like `snode_changes` can remain as it is. I don't think there are any special cases for notification nodes.